deployment/common/changeset/internal/ops/ops_mcms_deploy.go

```diff
@@ -0,0 +1,162 @@
+package ops
+
+import (
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	bindings "github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	evmMcms "github.com/smartcontractkit/mcms/sdk/evm"
+	mcmsTypes "github.com/smartcontractkit/mcms/types"
+
+	cldf_evm "github.com/smartcontractkit/chainlink-deployments-framework/chain/evm"
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+
+	"github.com/smartcontractkit/chainlink-deployments-framework/operations"
+	mcmsnew_zksync "github.com/smartcontractkit/chainlink/deployment/common/changeset/internal/evm/zksync"
+)
+
+type OpEVMMCMSDeps struct {
+	Chain    cldf_evm.Chain
+	Backend  bind.ContractBackend
```

Comment:
Same comment here about the backend, we can use cldf_evm.Chain for all instances of this

Reply:
Fixed